.pre-commit-config.yaml

```diff
@@ -4,17 +4,17 @@
 
 repos:
   - repo: https://github.com/psf/black
-    rev: stable
+    rev: 20.8b1
     hooks:
       - id: black
         language_version: python3.6
```

Comment:
Do we want to drop language version as in `packit-service`?

Reply:
yes, please